public/app/plugins/datasource/elasticsearch/hooks/useNextId.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('useNextId', () => {
       return (
         <ElasticsearchProvider
           query={query}
-          datasource={{} as any}
+          datasource={{}}
```

Comment:
```suggestion
          datasource={{} as ElasticDatasource}
```

Typescript is complaining about `{}` not being `ElasticDatasource`. To fix this, we can import `import { ElasticDatasource } from '../datasource';` and use `datasource={{} as ElasticDatasource}` in this case.